### PR TITLE
Updated NPSP Translations

### PR DESCRIPTION
**Pull request by force2b**
# Critical Changes

# Changes

- Updated Translations for all supported languages:
  - Field Labels
  - Field Help Text
  - Object Labels
  - Picklist Labels
  - Custom Buttons and Links
  - Custom Labels
  - Tab Labels
  - Packaged Page Layout Section Titles

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by force2b** [comment]
@GloryChoe Would you mind doing a quick review of this PR? This is a rebuild of the translations for Cumulus from the `feature/ngohana/2019.07.localization` branch, which I believe is the latest working version. I used the process we've been discussing where the translations are pushed into a scratch org and then retrieved back so that they're _properly_ formatted. Some additional cleanup is needed to remove extraneous elements returned by that process, but otherwise I think this looks pretty solid.
CC @mpusto

File: src/objectTranslations/Account-de.objectTranslation [hunk 1]
```diff
@@ -1,13 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
-    <fields>
-        <help>Der Batch, in dem dieser Account erstellt wurde</help>
-        <label><!-- Batch --></label>
+    <fieldSets>
+        <label><!-- BDE Entry FS --></label>
```
Comment (GloryChoe) [inline src/objectTranslations/Account-de.objectTranslation]: @force2b QQ, Michael, when the translations are entered in like this with , how does it show up in the actual UI? Are these labels user-facing? Will the users see the labels in English like this in the product?

Comment (force2b) [reply src/objectTranslations/Account-de.objectTranslation]: Good question. This is actually how Salesforce's defaults all untranslated items in metadata. So, in this case the fieldset for "BDE Entry FS" shows as untranslated (i.e., English) in the UI.

File: src/objectTranslations/Contact-es.objectTranslation [hunk 2]
```diff
@@ -1,129 +1,138 @@
...
-        <label>Cantidad de créd. blandos últimos N días</label>
+        <help>El número de créditos blandos atribuidos a este contacto que se cerraron en los últimos N días. El valor de N se define en Configuración de NPSP | Donaciones | Datos estadísticos de donantes.</help>
+        <label>Número créd. blandos en últimos N días</label>
         <name>Number_of_Soft_Credits_Last_N_Days__c</name>
     </fields>
     <fields>
-        <help>Cantidad de créditos blandos el año pasado</help>
-        <label>Cantidad de créd. blandos año pasado</label>
+        <help>El número de créditos blandos atribuidos a este contacto en el año de calendario anterior.</help>
+        <label>Número de créditos blandos el año pasado</label>
```
Comment (GloryChoe) [inline src/objectTranslations/Contact-es.objectTranslation]: @force2b Do you happen to know what this label looks like in the UI? I wonder if the previous translation used the abbreviation intentionally for creditos bcuz there's some limited space in the UI.

Comment (force2b) [reply src/objectTranslations/Contact-es.objectTranslation]: Just so I'm sure, are you referring to the label or the help text on the `Number_of_Soft_Credits_Last_Year__c` field?

Comment (GloryChoe) [reply src/objectTranslations/Contact-es.objectTranslation]: "label" - it was previously abbreviated to "cred." instead of "creditos (new translation)"

Comment (force2b) [reply src/objectTranslations/Contact-es.objectTranslation]: @GloryChoe How is this screen capture:
image

File: src/objectTranslations/Contact-es.objectTranslation [hunk 3]
```diff
@@ -1,129 +1,138 @@
...
-        <help>Fecha del último crédito blando</help>
+        <help>La fecha del último crédito blando de este contacto.</help>
         <label>Fecha del último crédito blando</label>
         <name>Last_Soft_Credit_Date__c</name>
     </fields>
     <fields>
-        <help>La cantidad de créditos blandos, cerradas en los últimos días definidos en la configuración del NPSP</help>
-        <label>Cantidad de créd. blandos últimos N días</label>
+        <help>El número de créditos blandos atribuidos a este contacto que se cerraron en los últimos N días. El valor de N se define en Configuración de NPSP | Donaciones | Datos estadísticos de donantes.</help>
+        <label>Número créd. blandos en últimos N días</label>
```
Comment (GloryChoe) [inline src/objectTranslations/Contact-es.objectTranslation]: @force2b To be consistent with the translations in other similar labels in this UI, I need to fix this to 
<label>Número créditos blandos en últimos N días</label>
What should I do to fix this or can you fix it in this PR?

Comment (force2b) [reply src/objectTranslations/Contact-es.objectTranslation]: If you're just changing the one label, you can simply edit in this PR. 

However I want to be sure this is the **final** PR that we're going to use for Cumulus. This PR is based on the original `feature/ngohana/2019.07.localization` that I created back in July from your first PR, and it has all of the new labels and fixes that we made back then to address the identified localization issues. If you're comfortable **this** is the PR we're going with (and will merge), you can use the Edit button on the `Contact-es.objectTranslation` file to change it (or I can make the label change if you prefer). Either way is fine as long as you're comfortable with PR.

File: src/objectTranslations/Engagement_Plan__c-fr.objectTranslation [hunk 4]
```diff
@@ -2,35 +2,35 @@
...
     <caseValues>
         <plural>true</plural>
-        <value><!-- Engagement Plan --></value>
+        <value>Plans d&apos;engagement</value>
     </caseValues>
     <fields>
         <label>Compte</label>
         <name>Account__c</name>
-        <relationshipLabel>Plans d’engagement</relationshipLabel>
+        <relationshipLabel><!-- Engagement Plans --></relationshipLabel>
```
Comment (GloryChoe) [inline src/objectTranslations/Engagement_Plan__c-fr.objectTranslation]: The translations got reverted here and for the same "relationshipLabel"s below as well. Is this expected? @force2b

Comment (force2b) [reply src/objectTranslations/Engagement_Plan__c-fr.objectTranslation]: Eagle Eyes!!! Great catch @GloryChoe! I confirm that this is missing in this PR and in another test one that generated using a slightly different technique. Looking into it this morning and will let you know what I find.

Comment (force2b) [reply src/objectTranslations/Engagement_Plan__c-fr.objectTranslation]: @GloryChoe After reiewing, I can see that the Related List translations for the Account__c, Contact__c, Case__c, Campaign__c, Opportunity__c and RecurringDonation__c fields are not in original PR that this work is based on. You should be able to see that in this file from the parent `feature/ngohana/2019.07.localization` branch: https://github.com/SalesforceFoundation/Cumulus/blob/feature/ngohana/2019.07.localization/src/objectTranslations/Engagement_Plan__c-fr.objectTranslation

The result is that there the related list label translations applied are not in the scratch org this they don't exist in the metadata, which is why they're not in what gets pulled back from the scratch org.
image

I searched through the PR and if I'm not mistaken this is the only object with missing Related List Label translations. I think the question here is why did the translation tool not include these.

Comment (GloryChoe) [reply src/objectTranslations/Engagement_Plan__c-fr.objectTranslation]: Hi @force2b - I've finished my review. All looks good except the reverted translations for Engagement Plans mentioned above. Thanks!

Comment (force2b) [reply src/objectTranslations/Engagement_Plan__c-fr.objectTranslation]: @GloryChoe The EngagementPlan issue should now be resolved via https://github.com/SalesforceFoundation/Cumulus/pull/4878/commits/8a11b367d69dc9a0955b2becc3ffa0e57e7cb4b5

File: src/objectTranslations/Address_Verification_Settings__c-de.objectTranslation [hunk 5]
```diff
@@ -42,57 +42,57 @@
...
         <name>Auth_Token__c</name>
     </fields>
     <fields>
         <help>Bei Aktivierung dieser Option werden geänderte Adressen automatisch mit der Antwort vom Service aktualisiert (falls ein überprüften Ergebnis vorhanden ist).</help>
-        <label><!-- Auto-Update Addresses-DEPRECATED --></label>
+        <label>Adressen autom. aktualisieren – ERSETZT</label>
         <name>Auto_Update_Addresses__c</name>
     </fields>
     <fields>
-        <help>Für die Kontaktierung der SmartlyStreets-API und Überprüfung der Adressen zuständige Salesforce-Klasse</help>
```
Comment (mpusto) [inline src/objectTranslations/Address_Verification_Settings__c-de.objectTranslation]: Shouldn't this be SmartyStreets? I am not sure how to edit directly

Comment (GloryChoe) [reply src/objectTranslations/Address_Verification_Settings__c-de.objectTranslation]: It is corrected to "smarty" in the new delivery. See green changes on the right side :)

**Comment by force2b** [comment]
**lurch: attach W-035813

**Comment by force2b** [comment]
**lurch: attach W-035813

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001CAKSQA4" target="_blank">W-035813</a>

**Pull request merged**